lib/topic_query.rb

```diff
@@ -269,10 +269,14 @@ def list_latest
   end
 
   def list_filter
-    results =
-      TopicsFilter.new(guardian: @guardian, scope: latest_results).filter_from_query_string(
-        @options[:q],
-      )
+    topics_filter =
+      TopicsFilter.new(guardian: @guardian, scope: latest_results(skip_remove_muted: true))
```

Comment:
`skip_remove_muted` reads like a double negative which is a bit hard to understand (to me). May I suggest `allow_muted` or `include_muted`?